assets/dev/js/editor/elements/views/container.js

```diff
@@ -359,7 +379,7 @@ const ContainerView = BaseElementView.extend( {
 		BaseElementView.prototype.renderOnChange.apply( this, arguments );
 
 		// Re-initialize the droppable in order to make sure the axis works properly.
-		if ( settings.changed.flex_direction ) {
+		if ( settings.changed.flex_direction || settings.changed.content_width ) {
 			this.$el.html5Droppable( 'destroy' );
 			this.$el.html5Droppable( this.getDroppableOptions() );
 		}
```

Comment:
Destroy both container and container inner.

Reply:
I have created new methods for destroy and initialisation. 
Destroying an undefined element gives an error. I have stored the initialisation in the element settings. In this way I can destroy the correct elements.

Reply:
Done in next code update.